apps/dashboard/src/main/java/com/akto/listener/InitializerListener.java

```diff
@@ -109,6 +110,51 @@ public void run() {
             }
         }, 0, 4, TimeUnit.HOURS);
     }
+    static void editTestSourceConfig() throws IOException{
+        try {
+            List<TestSourceConfig> detailsTest = TestSourceConfigsDao.instance.findAll(new BasicDBObject()) ;
+            for(TestSourceConfig tsc : detailsTest){
+                String filePath = tsc.getId() ;
+                filePath = filePath.replace("https://github.com/", "https://raw.githubusercontent.com/").replace("/blob/", "/");
+                try {
+                    FileUtils.copyURLToFile(new URL(filePath), new File(filePath));
+                } catch (IOException e1) {
+                    e1.printStackTrace();
+                    continue;
+                }
+
+                Yaml yaml = new Yaml();
+                InputStream inputStream = java.nio.file.Files.newInputStream(new File(filePath).toPath());
+                Map<String, Map<String,Object>> data = yaml.load(inputStream);
+                if (data == null) data = new HashMap<>();
+
+                Map<String ,Object> currObj = data.get("info");
+                String description = (String) currObj.get("name");
+                tsc.setDescription(description);
+
+                String severity = (String) currObj.get("severity");
+                Severity castedSeverity = Severity.LOW ;
+                if(severity.toLowerCase() != "unknown"){
+                    castedSeverity = Severity.valueOf(severity.toUpperCase()) ;
+                    tsc.setSeverity(castedSeverity);
+                }
+                
+                String stringTags = (String) currObj.get("tags") ;
+                List<String> tags = new ArrayList<>();
+                if(stringTags != null){
+                    tags = Arrays.asList(stringTags.split(","));
+		            tsc.setTags(tags);
+                }
+                
+
+                TestSourceConfigsDao.instance.updateOne(Filters.eq("_id", tsc.getId()),
+                        Updates.combine(Updates.set("description", description),
+                                Updates.set("severity", castedSeverity), Updates.set("tags", tags)));
```

Comment:
would have preferred bulk update. but since the collection has too few documents, let's keep this single update

Reply:
I think bulk updates do same changes for all tests, but as tags, severity and description, that's why i tried single update in loop.